### ISSUE TITLE
listener_kwargs in "prices_file_to_data_frame()" 

### ISSUE THREAD

**Issue by nachouve** [mberk/betfairutil#19]
Would be possible pass some arguments to the listener of the function prices_file_to_data_frame()?

It would be great to pass a listener_kwargs dict to get a subset of prices. 
Something similar to https://betcode-org.github.io/flumine/quickstart/#listener-kwargs

**Comment by mberk** [comment]
Yes, happy to add this. It wouldn't really be `listener_kwargs` per se as these get actioned by flumine not betfairlightweight. With that in mind, anything else you would like to be able to filter other than inplay/seconds to market start?

**Comment by nachouve** [comment]
I have tried to implement that filter myself. I saw that and gave up....
But if you skip all the unwanted lines before transforming them into python objects, it speeds up some analysis I do.

About more filters... Maybe filtering only one prices side or some specific selection_ids could be good too!

**Comment by mberk** [comment]
I think these are all questions for @liampauling re: betfairlightweight. I would only be applying the filtering after having generated all of the market books. If you want increased performance then #18 is your best bet

**Comment by liampauling** [comment]
With some extra code `listener_kwargs` can be passed down to bflw now however as Mo mentioned this is now redundant due to #18
